packages/next/src/migrations/update-9-2-0/create-next-config.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { Tree } from '@angular-devkit/schematics';
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+import * as path from 'path';
+import { createEmptyWorkspace } from '@nrwl/workspace/testing';
+
+describe('create-next-config-9.2.0', () => {
+  let tree: Tree;
+  let schematicRunner: SchematicTestRunner;
+
+  beforeEach(async () => {
+    tree = Tree.empty();
+    tree = createEmptyWorkspace(tree);
+    schematicRunner = new SchematicTestRunner(
+      '@nrwl/next',
+      path.join(__dirname, '../../../migrations.json')
+    );
+  });
+
+  it('should create next.config.js if it does not exist', async () => {
+    tree.overwrite(
+      '/workspace.json',
+      JSON.stringify({
+        version: 1,
+        projects: {
+          demo: {
```

Comment:
Is it worth adding some dummy projects just to prove that the migration will actually skip over them? Maybe some projects without builders as well?

e.g.

```ts
          notnext: {
            root: 'apps/notnext',
            sourceRoot: 'apps/notnext/src',
            architect: {
              build: {
                builder: '@nrwl/web:build',
                options: {}
              }
            }
          },
          nobuilder: {
            root: 'apps/nobuilder',
            sourceRoot: 'apps/nobuilder/src',
            architect: {}
          }
```